pilot/pkg/bootstrap/certcontroller.go

```diff
@@ -49,13 +49,19 @@ const (
 )
 
 // initDNSCerts will create the certificates to be used by Istiod GRPC server and webhooks.
+// Only called if ./var/run/secrets/istiod/tls is not set - which has priority.
```

Comment:
THis is not quite correct, see `hasCustomTLSCerts`

Reply:
Which part is not correct ? You mean hasCustomTLSCerts can also use the custom env variable for a different location, or that 'if all files are present in ../istiod/tls' ?

How about: only called if 'hasCustomTLSCert' does not find the certs in /var/run/.../istio/tls or custom location ?

I think it's useful to have the common default location in the comments, it takes effort to navigate 3 layers of constants.

Reply:
>You mean hasCustomTLSCerts can also use the custom env variable for a different location,

Yes, the path can be any dir

>How about: only called if 'hasCustomTLSCert' does not find the certs in /var/run/.../istio/tls or custom location ?

LGTM